normandy/control/static/control/js/actions/ControlActions.js

```diff
@@ -98,11 +105,14 @@ function requestInProgress() {
   };
 }
 
-function requestComplete(status) {
-  return {
-    type: REQUEST_COMPLETE,
-    status: status
-  };
+function requestComplete(result) {
+  return (dispatch) => {
```

Comment:
Also @potch just mentioned Airbnb released some ES6 style guides that can hook into eslint that we might want to consier.